thriftserver/thriftserver/overlays/create-spark-cluster/spark-cluster.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: radanalytics.io/v1
+kind: SparkCluster
+metadata:
+  name: spark-cluster-thriftserver
+spec:
```

Comment:
I am not sure what the defaults are for resource requests/limits and I could not find it in the Spark Operator repo (cc @tmckayus - any ideas?). Maybe memory and cpus should be explicitly stated here?

Reply:
@vpavlin there are no default values for memory/cpu requests and limits, the pods will run with whatever openshift defaults are in effect if they are not specified.  If the "memoryLimit/memoryRequest" and/or "cpuLimit/cpuRequest" fields are specified in the CR, they will be used to set fields in the pod spec.

(also note the deprecated "memory" and "cpu" fields which set limit and request to the same value, but the new forms should be used)

Reply:
@vpavlin sure, we can state some defaults in there. Would you agree that, since this is like the minimal cluster for "demo" and "test" purposes, we can go with some reasonable minimal values which are sufficient to run the cluster without much workload expected? I think, once the user really wants to use this component, they would model the worker counts, resource requirements etc to their specific need.

Reply:
My main concern was to make sure it matches whatever is expected here https://github.com/opendatahub-io/odh-manifests/blob/3d2bba5ad161f6ffb8e5de84cca8a0e4a91f6731/thriftserver/thriftserver/base/thriftserver-server-conf-secret.yaml